storage/migrations/02_runtimes.up.sql

```diff
@@ -48,11 +48,19 @@ CREATE TABLE chain.runtime_transactions
   size UINT31 NOT NULL,
   
   -- Transaction contents.
-  method      TEXT,         -- accounts.Transter, consensus.Deposit, consensus.Withdraw, evm.Create, evm.Call. NULL for malformed and encrypted txs.
+  method      TEXT,         -- accounts.Transfer, consensus.Deposit, consensus.Withdraw, evm.Create, evm.Call. NULL for malformed and encrypted txs.
   body        JSONB,        -- For EVM txs, the EVM method and args are encoded in here. NULL for malformed and encrypted txs.
   "to"        oasis_addr,   -- Exact semantics depend on method. Extracted from body; for convenience only.
   amount      UINT_NUMERIC, -- Exact semantics depend on method. Extracted from body; for convenience only.
 
+  -- Encrypted data in encrypted Ethereum-format transactions.
+  evm_encrypted_format INTEGER,
+  evm_encrypted_public_key BYTEA,
+  evm_encrypted_data_nonce BYTEA,
+  evm_encrypted_data_data BYTEA,
+  evm_encrypted_result_nonce BYTEA,
+  evm_encrypted_result_data BYTEA,
```

Comment:
Hm ... IDK how much of this we should store in the DB. Especially storing `data_data` and `result_data` will probably about double the size of each row?
@kostko what was your thinking on what we should make easily accessible to the users? Just the encryption format (including NULL, importantly), or more? The encryption public key is [ephemeral](https://github.com/oasisprotocol/oasis-sdk/blob/c823194ed60ad7a922812e6b803d8ff8b9f78c45/client-sdk/go/types/callformat.go#L7-L7), and the nonce is non-sequential, so neither sounds that useful. The encrypted data is only useful to someone who knows how to decode it locally; I have no idea if we expect users of that caliber, and if we want to support them beyond giving them a public gRPC node.

Reply:
yup needs discussion

Reply:
After today's product sync: Nikola thinks we should show everything. Let's go ahead with this as-is, and we can prune the DB columns later if needed.

Reply:
kept..........................................................................